src/components/Drawer/readme.md

```diff
@@ -0,0 +1,327 @@
+##### Drawer example:
+
+```js
+import React from 'react';
+import { Button, Drawer } from 'react-rainbow-components';
+
+const initialState = {
+    isOpenTop: false,
+    isOpenRight: false,
+    isOpenBottom: false,
+    isOpenLeft: false,
+};
+
+<div
+    className="rainbow-m-vertical_large rainbow-p-horizontal_small rainbow-m_auto rainbow-flex_wrap"
+>
+    <div className="rainbow-align-content_center rainbow-p-medium">
+        <Button
+            disabled
+            className="rainbow-m-around_medium"
+            label="Open Top"
+            onClick={() => setState({
+                isOpenTop: true,
+            })}
+        />
+    </div>
+    <div className="rainbow-flex rainbow-flex_row">
+        <div className="rainbow-align-content_center rainbow-p-medium rainbow-m_auto">
+            <Button
+                className="rainbow-m-around_medium"
+                label="Open Left"
+                onClick={() => setState({
+                    isOpenLeft: true,
+                })}
+            />
+        </div>
+        <div className="rainbow-align-content_center rainbow-p-medium rainbow-m_auto">
+            <Button
+                className="rainbow-m-around_medium"
+                label="Open Right"
+                onClick={() => setState({
+                    isOpenRight: true,
+                })}
+            />
+        </div>
+    </div>
+    <div className="rainbow-align-content_center rainbow-p-medium">
+        <Button
+            disabled
+            className="rainbow-m-around_medium"
+            label="Open Bottom"
+            onClick={() => setState({
+                isOpenBottom: true,
+            })}
+        />
+    </div>
+    <Drawer
+        id="drawer-right-1"
+        header="This is a drawer"
+        isOpen={state.isOpenRight}
+        onRequestClose={() => setState({ isOpenRight : false })}
+    />
+    <Drawer
+        id="drawer-left-1"
+        header="This is a drawer"
+        isOpen={state.isOpenLeft}
+        slideFrom="left"
```

Comment:
when passing a wrong value of slideFrom the component should render with its default value: 'left'